components/article-list-item.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Link from 'next/link';
+import { string } from 'prop-types';
+
+const Item = (props) =>{
+  return (
+    <div className="card">
+      <Link href="/a/[slug]" as={`/a/${props.slug}`}>
+        <div className="columns">
+          <div className="column is-half">
+            <figure className="image">
+              <img src={props.thumbnail} alt="Placeholder image" />
+            </figure>
+          </div>
+          <div className="card-content">
+            <h3 className="title is-5">{ props.title }</h3>
+            <div className="article-meta">
+              <span>Category: </span><strong>{ props.category }</strong>
+              <span>Author: </span><strong>{ props.author }</strong>
+            </div>
+            <div className="content">
+              <p>{ props.excerpt }</p>
+            </div>
+            <div className="level">
+              <div className="level-left"></div>
+              <div className="level-right">
+                <div className="level-item">
+                  <a href="/" className="readmore">
+                    read more...
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </Link>
+      <style jsx>{`
```

Comment:
If you want to go with CSS in JS, I would suggest going for [styled-components](https://styled-components.com/). They are automatically scoped, make it very easy to turn elements reusable, they [accept props you can then process](https://styled-components.com/docs/basics#passed-props), allow you to create [themes](https://styled-components.com/docs/advanced#theming) and [mixins](https://tobbelindstrom.com/blog/how-to-create-a-breakpoint-mixin-with-styled-components/) (because it's just JavaScript).

If you decide on using them, remember that you need some [custom stuff for it to work in Next.js](https://styled-components.com/docs/advanced#nextjs) (but it's worth it).